docs/components/tooltip.md

```diff
@@ -63,6 +63,10 @@ Here are the different options:
 </KoolTip>
 ```
 
+### positionFixed
```

Comment:
Again, unsure if props should be `camelCase` or `PascalCase` ?

Reply:
Set all props to `camelCase`!